### PR TITLE
refs task #116927 Fix-CI: calendars.scss

### PR DESCRIPTION
**Pull request by mdmahmudurrahman**
## Input

```
scss-lint --require=scss_lint_reporter_checkstyle --format=Checkstyle app/assets/stylesheets/calendars.scss
```
## Output

```
<?xml version="1.0" encoding="utf-8"?>
<checkstyle version="1.5.6">
</checkstyle>
```

### CODE REVIEW

File: app/assets/stylesheets/calendars.scss [hunk 1]
```diff
@@ -1,81 +1,56 @@
...
     position: relative;
   }
+
   .mainbody {
     margin-left: 184px;
     margin-right: 0;
     margin-top: 10px;
-    -webkit-transition: margin-left .218s;
-    -moz-transition: margin-left .218s;
-    #full-calendar {
```
Comment (dieunb) [inline app/assets/stylesheets/calendars.scss]: why do you remove it?

File: app/assets/stylesheets/calendars.scss [hunk 2]
```diff
@@ -1,81 +1,56 @@
...
+    right: 5%;
   }
+
   h2 {
     display: block;
     font-size: 1.5em;
     font-weight: bold;
     margin: 0;
   }
-  .fc-button {
```
Comment (dieunb) [inline app/assets/stylesheets/calendars.scss]: why do you remove it?

File: app/assets/stylesheets/calendars.scss [hunk 3]
```diff
@@ -84,46 +59,72 @@
...
   .data-none-display {
     display: none;
   }
+
   .data-display {
     display: inline-block;
   }
-  #current-user-id-popup {
+
+  .close-popup {
```
Comment (dieunb) [inline app/assets/stylesheets/calendars.scss]: i have just removed this

File: app/views/events/_popup.html.erb [hunk 4]
```diff
@@ -17,7 +17,7 @@
           <div id="data-popup">
             <div id="time-data" class="form-group">
               <label><%= t ".time" %></label>
-              <span id="time-event-popup" class="title-content">
```
Comment (dieunb) [inline app/views/events/_popup.html.erb]: this line should keep

Comment (dieunb) [inline app/views/events/_popup.html.erb]: this line should keep

File: app/assets/stylesheets/calendars.scss [hunk 5]
```diff
@@ -84,46 +59,72 @@
...
-    width: 335px;
     height: 20px;
+    width: 335px;
   }
+
   .title-content {
     font-size: 13px;
   }
+
+  .time-event-popup-title {
```
Comment (dieunb) [inline app/assets/stylesheets/calendars.scss]: remove it and all where you defined. I have used `title-content` instead

File: app/assets/stylesheets/calendars.scss (hunk 2 above)
Comment (mdmahmudurrahman) [inline app/assets/stylesheets/calendars.scss]: @dieunb I didn't find any use of this block in the calendars view. (bow)

File: app/views/events/_popup.html.erb [hunk 6]
```diff
@@ -17,7 +17,7 @@
...
+              <span id="time-event-popup" class="tile-content">
```
Comment (dieunb) [inline app/views/events/_popup.html.erb]: `title-content` not `tile-content`

File: app/views/events/show.html.erb [hunk 7]
```diff
@@ -8,7 +8,7 @@
     <div class="lb-dt">
       <div class="ev-dt">
         <label><%= t ".time" %></label>
-        <span id="time-event-popup" class="title-content">
+        <span id="time-event-popup" class="time-event-popup-title tile-content">
```
Comment (dieunb) [inline app/views/events/show.html.erb]: - `title-content` not `tile-content`
- remove `time-event-popup-title`

File: app/assets/stylesheets/calendars.scss (hunk 1 above)
Comment (mdmahmudurrahman) [inline app/assets/stylesheets/calendars.scss]: @dieunb I replaced it with a class. (bow)

File: app/assets/stylesheets/calendars.scss (hunk 2 above)
Comment (dieunb) [inline app/assets/stylesheets/calendars.scss]: This class will generate by fullcalendar js lib. Show i think we can ignore it from CI

File: app/assets/stylesheets/calendars.scss [hunk 8]
```diff
@@ -1,81 +1,68 @@
...
+      width: 15%;
+    }
+
+    .fc-head,
+    .fc-axis {
+      color: $emperor-approx;
+      font-size: 1em;
     }
   }
-  a.fc-time-grid-event.fc-v-event.fc-event.fc-start.fc-end.color-9.fc-draggable.fc-resizable {
```
Comment (dieunb) [inline app/assets/stylesheets/calendars.scss]: This class will generate by fullcalendar js lib. Show i think we can ignore it from CI

File: app/assets/stylesheets/calendars.scss (hunk 1 above)
Comment (dieunb) [inline app/assets/stylesheets/calendars.scss]: This class will generate by fullcalendar js lib. Show i think we can ignore it from CI

**Pull request merged**